### PR TITLE
Artist layout social paddding/margin changes

### PR DESCRIPTION
**Pull request by msbodetti**
### References Issues:
https://app.asana.com/0/1141185905028320/1148999973930772/f
### Release Screenshots / Video:
https://share.getcloudapp.com/8LuwGmdR
https://share.getcloudapp.com/jkungNr4
### Environment Variables:
 * No change

### CODE REVIEW

File: src/components/elements/event/ArtistSummary.js [hunk 1]
```diff
@@ -115,88 +115,76 @@ class ArtistSummary extends Component {
...
-					<Grid item xs={2}>
-						<SocialIconLink
-							icon={"facebook"}
-							userName={facebook_username}
-							size={40}
-							color={"neon"}
-						/>
-					</Grid>
+					<SocialIconLink
+						style={{ marginRight: 10 }}
```
Comment (Jasonvdb) [inline src/components/elements/event/ArtistSummary.js]: If a style is used multiple times it's best to add a classname. Might have to change the `SocialIconLink` component to accept a sting prop called `className` and then use `classnames` function to apply both classes to the container element

Comment (msbodetti) [reply src/components/elements/event/ArtistSummary.js]: Thanks @Jasonvdb. I've updated it.

File: src/components/elements/social/SocialIconLink.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ const SocialIconLink = props => {
 
 	return (
 		<a
-			className={classes.container}
+			className={`${classes.container} ${className}`}
```
Comment (Jasonvdb) [inline src/components/elements/social/SocialIconLink.js]: Rather go with this import:

`import classnames from "classnames"`

And then use it like this:

```
className={classnames({
	[classes. container]: true,
	[className]: !!className
})}

```

File: src/components/elements/social/SocialIconLink.js [hunk 3]
```diff
@@ -100,6 +100,7 @@ SocialIconLink.defaultProps = {
 
 SocialIconLink.propTypes = {
 	classes: PropTypes.object.isRequired,
+	className: PropTypes.object,
```
Comment (Jasonvdb) [inline src/components/elements/social/SocialIconLink.js]: Should be a string?

Comment (msbodetti) [reply src/components/elements/social/SocialIconLink.js]: Thanks @Jasonvdb have made the updates!

**Pull request merged**